src/libs/workspace-utils.test.ts

```diff
@@ -132,14 +138,73 @@ describe('isValidWsExportTarget', () => {
   });
 });
 
-describe('hasProtectedData', () => {
```

Comment:
`hasProtectedData` (which was Azure-specific) has been deleted in favor of `isProtectedWorkspace`.